tests/spec/dirty_tests.py

```diff
@@ -50,7 +50,7 @@ class DirtyTest(object):
 \\S+ +\\d+ +\\d+ +\\d+ +\\d+ +\\d+ +\\S+ +\\S+ +\\S+ +\\S+ +{cmd}
 
 There are two ways you can fix this:
-  \\* temporarily: lsof -t playground/{service} | xargs kill -9
+  \\* temporarily: pgctl-fuser playground/{service} | xargs kill -9
   \\* permanently: http://pgctl.readthedocs.org/en/latest/user/quickstart.html#writing-playground-services
 
```

Comment:
Keep the link to the wiki actually. It's where I explain how to fix this problem "for real".